tests/test_workers/test_tasks/test_opm_operations.py

```diff
@@ -754,3 +754,12 @@ def test_verify_operator_exists(
     )
     mock_ors.assert_has_calls([mock.call(db_path=index_db_path)])
     assert package_exists == opr_exists
+
+
+@mock.patch('iib.workers.tasks.utils.run_cmd')
+def test_opm_validate(mock_rc, tmpdir):
```

Comment:
I am not sure if this test is meaningful. 
It is just testing if `run_cmd` really run the command. 

I would suggest to move this test to `test_merge_catalogs_dirs` where it makes sense to test if `opm validate` was called.

But we can keep this one as well. But it would be nice to test it in the `test_merge_catalogs_dirs`

Reply:
Ok, I've moved this check to the `test_merge_catalog_dirs`, I also agree it makes more sense